### PR TITLE
Header

### LINKED ISSUE #1: Build a great header

### PR DESCRIPTION
**Pull request by anitaschiller**
I added the styling for the header, please review!

Fixes #1

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+header h1 {
+  color: powderblue;
+  padding: 15px;
+  text-shadow: -1px 0 hotpink, 0 1px hotpink, 1px 0 hotpink, 0 -1px hotpink;
+}
+
+h2 {
+  margin: 0;
+  margin: 50px 0;
+  border: 2px solid black;
```
Comment (ThomasKirst) [inline styles.css:23]: Maybe a little padding to the left would give the h2 some room to breathe.

**Review by ThomasKirst — CHANGES_REQUESTED** [review]
Overall great work, I like the colour combinations.
I have only a few remarks.

File: styles.css (hunk 1 above)
Comment (anitaschiller) [reply styles.css:23]: Good catch, done!

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+header h1 {
+  color: powderblue;
+  padding: 1rem;
+  text-shadow: -1px 0 hotpink, 0 1px hotpink, 1px 0 hotpink, 0 -1px hotpink;
+}
+
+h2 {
+  margin: 0;
```
Comment (anitaschiller) [inline styles.css:21]:
```suggestion
  margin: 20px;
```

**Review by ThomasKirst — APPROVED** [review]
LGTM!

**Pull request merged**